Gir.toml

```diff
@@ -413,6 +417,15 @@ name = "Gdk.Monitor"
 status = "generate"
 generate_builder = true
 
+[[object]]
+name = "Gdk.PopupLayout"
+status = "generate"
+manual_traits = ["PopupLayoutExtManual"]
+    [[object.function]]
+    name = "get_offset"
+    # invalid mutability, manual
```

Comment:
`const = true` on the parameter in question doesn't help?

Reply:
It doesn't help here neither, though I forgot to add the manual get_offset. Should be done

Reply:
Needs to get `(out)` annotations for these two parameters.